app/src/main/java/learning/android/tenmarks/com/androidlearning/DashBoardActivity.java

```diff
@@ -226,4 +227,11 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
         return super.onOptionsItemSelected(item);
     }
+
+    @Override
+    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+
+        Toast.makeText(DashBoardActivity.this, resultCode + " : " + data.getStringExtra("return_value"), Toast.LENGTH_SHORT).show();
```

Comment:
It's always good to check if data is not null before you access it. 
Also if a key exists.
For some types you can get a default value like data.getIntExtra("my_value", 1); in case the key is not found.